spec/integration.rb

```diff
@@ -318,6 +318,11 @@
                             'install --no-repo-update --no-verbose'
     end
 
+    describe 'Integrates a Pod using a vendored static xcframework' do
```

Comment:
looks like a test cannot be found for this. You can add a unit test also which should be sufficient. Look into `embed_fraameworks_script_spec.rb` file.

Reply:
adding such an example though should be good as well.

Reply:
There's a PR open on the integration tests repo here: https://github.com/CocoaPods/cocoapods-integration-specs/pull/289

Reply:
ah my bad!